Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
+ruby '2.0.0'
```

Comment:
Scratch that--specs fail with anything newer than 2.0.0 :( so i think we need to stay with 2.0.0 for now until refactoring happens